app/presenters/welcome/welcome_presenter.rb

```diff
@@ -0,0 +1,15 @@
+module Welcome
+  class WelcomePresenter
+    def attendee_count
+      Attendee.count
```

Comment:
Are we getting a count of ALL the attendees in our db? or attendees for some SPECIFIC events?